catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/TestSolrProvider.java

```diff
@@ -1840,7 +1840,7 @@ public void testQueryMissingSortField() throws IngestException, UnsupportedQuery
 
         QueryImpl query = new QueryImpl(txtFilter);
 
-        query.setSortBy(new ddf.catalog.filter.SortByImpl("unknownField", SortOrder.ASCENDING));
+        query.setSortBy(new ddf.catalog.filter.impl.SortByImpl("unknownField", SortOrder.ASCENDING));
```

Comment:
Probably not, but there were quite a few of them and I would have had to do it manually so I just did a find/replace.